collections/sort_by.ts

```diff
@@ -64,7 +97,7 @@ export function sortBy<T>(
     const b = selectors[bi];
     if (a === null) return 1;
     if (b === null) return -1;
```

Comment:
Do these lines also need to be changed?

How about swapping `a` and `b` above if the order is "desc"? (In that case I think we don't need to switch `comparator`)

Reply:
Now they are swapped, good catch! Also the idea of `comparator` is to extract the function ordering once and call it when `a` and `b` are not null. However I do not understand why coverage says there's no coverage in `comparator` 🤔